validphys2/src/validphys/results.py

```diff
@@ -224,6 +224,25 @@ def procs_data_values(proc_result_table):
     data_central_values = proc_result_table["data_central"]
     return data_central_values
 
+def commondata_wc(data):
+    """
+    commondata with cuts:
+    given a DataGroupSpec load all the DataSetInput
+    instances into CommonData instances with cuts
+    already applied to
+
+    Parameters
+    ----------
+
+    data : validphys.core.DataGroupSpec
+    
+    Returns
+    -------
+    list containing commondata instances with cuts
+    
+    """
+    return data.load_commondata_instance()
```

Comment:
I'm very confused about this function, there is nothing enforcing the cuts here, right? How is this different from simply doing `data.load_commondata_instance()` inside `make_level1_data`?

Maybe there's something I've missed along the way?

Reply:
Hi @scarlehoff, I think you are right. This is completely unnecessary. I changed it in the new commits